# Open Data Accelerates Nobel Prize-Winning AI in Molecular Research

Within every cell of every living organism on the planet, almost all biological functions are carried out and catalysed by protein molecules. The complex structure of each of these proteins is blueprinted in DNA and can be thought of as a long chain of small amino acid molecules linked together. However, unlike a bicycle chain, these chains fold, twist and wrap around themselves, making each protein uniquely shaped to suit its tasks and functions.

Bioinformaticians are trying to identify the shapes of as many proteins as possible, to understand how structure facilitates function. Armed with this information, laboratories could [design, synthesise, and then mass-produce proteins to potentially revolutionise biomedical engineering](https://www.creative-proteomics.com/resource/protein-sequencing-significance-methods-applications.htm).

However, it is extremely difficult to study the structure of proteins. It requires researchers to identify how each amino acid in the protein chain interacts with one another depending on their sequence, which requires [massive innovations in microscopy, crystallography and diffraction](https://f1000research.com/articles/9-667/v1). The identification of protein structures has historically been a slow process. 

[AlphaFold](https://deepmind.google/technologies/alphafold/), the deep-learning AI model developed by Google DeepMind, completely shifted this paradigm. Upon its [introduction in 2018](https://ccsp.hms.harvard.edu/wp-content/uploads/2020/11/AlphaFold-at-CASP13-AlQuraishi.pdf), AlphaFold was a high-performing algorithm that could be used to computationally predict the shapes of proteins from their underlying amino acid sequence, although it was thought to be too inaccurate. Then, in 2020, the developer team released the groundbreaking AlphaFold 2, which [integrated elements from the field of AI Large Language Models](https://www.nature.com/articles/s41586-021-03819-2) into the algorithmic design of AlphaFold, outcompeting other computational methods in the field of bioinformatics and [recognised as a technological marvel.](https://www.nature.com/articles/d41586-020-03348-4)

In 2024, AlphaFold 3 was released. With some refinements and further training, this version exhibited [50-100% better performance than AlphaFold 2](https://www.chemh.com/beyond-alphafold-3-navigating-future-challenges-in-protein-structure-prediction/) across evaluation categories, firmly cementing the model as a necessary tool for the future identification and exploration of protein structures. AlphaFold 3 was such a technological breakthrough that two Google DeepMind executives won the [Nobel Prize for Chemistry in 2024](https://www.chemh.com/beyond-alphafold-3-navigating-future-challenges-in-protein-structure-prediction/). 

AlphaFold promises tangible impacts for the health of the entire world. It has brought much-needed [efficiency to molecular biology](https://www.creative-proteomics.com/resource/protein-sequencing-significance-methods-applications.htm), enabling [enhanced cancer drug discovery](https://pubs.rsc.org/en/content/articlelanding/2023/sc/d2sc05709c), providing [a fast route towards emergency medicine in pandemics](https://pmc.ncbi.nlm.nih.gov/articles/PMC8186955/), and aiding [the development of malaria vaccines](https://deepmind.google/discover/blog/stopping-malaria-in-its-tracks/?_gl=1*1abjk37*_up*MQ..*_ga*MTM2ODgzNjc3MC4xNzM2MjY5NDk3*_ga_LS8HVHCNQ0*MTczNjI2OTQ5Ni4xLjAuMTczNjI2OTQ5Ni4wLjAuMA..). It is an important next step in improving healthcare for millions of people around the world. 

## Open data for groundbreaking bioinformatics research
AlphaFold won awards for its innovative design, but it is important to note that it would not exist without the strong history of open data in bioinformatics. 

In 1971, bioinformatics researchers from around the world, conducting a growing number of experimental observations of proteins, needed infrastructure to allow them to easily share findings and visualise them in 3D. To enable better collaboration, faster peer-review, and overall efficiency in their field, [Cambridge Crystallographic Data Centre (UK)](https://www.ccdc.cam.ac.uk/) and [Brookhaven National Laboratory (US)](https://www.bnl.gov/world/) came together to develop the [Protein Data Bank (PDB)](https://www.rcsb.org/).

[Over the next 50 years](https://www.rcsb.org/stats/growth/growth-released-structures), almost every discovered protein sequence and molecular structure was openly published to the PDB. This massive database was what enabled AlphaFold’s development; as training data, each protein on the PDB could be used to teach the algorithmic model how proteins work, what sequences typically look like, and how sequences relate to structures. Without the mature open data infrastructure provided by the PDB (and [the 230,000 proteins openly published on it](https://www.rcsb.org/stats/growth/growth-released-structures)), the AlphaFold team would find it much more difficult to amass data to train their models to the groundbreaking extent that they have achieved.

AlphaFold returns the favour: every protein structure predicted by the AlphaFold model is publicly released and openly published to the [AlphaFold Protein Structure Database](https://alphafold.ebi.ac.uk/). This database, in its latest release, contains more than two million entries, each of which can be verified or falsified with practical experiments by bioinformaticians.

## About the data
Proteins are complex, and their data is perhaps even more so. A single observation of [human insulin on the PDB](https://www.rcsb.org/structure/1MSO) provides information on: its classification; the organism it was collected from and the experimental organism used as an expression system; whether the protein is a mutation of another; the collection experiment’s results and external validation metrics; the literature the experiment is documented in; details about the macromolecule; the sequence of amino acids that makes up the protein; and an interactive 3D representation of the protein’s physical structure. 

All of this data can be downloaded from the PDB, either at the click of a button or with an API call, meaning it is not only extremely in-depth but also accessible to those who need it. Furthermore, because all data files published to the archive are made available under the [CC0 1.0 Universal Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/deed.en), there are no restrictions on its use.

The AlphaFold Protein Structure Database also publishes with the CC0 dedication. The [human insulin entry](https://alphafold.ebi.ac.uk/entry/P01308) on this database is accompanied by AlphaFold’s prediction of the protein’s sequence, the model’s confidence in its prediction, and an interactive 3D explorer for the predicted structure.

Also made openly available on the AlphaFold database is the [AlphaMissense](https://www.science.org/doi/10.1126/science.adg7492) assessment of each protein. AlphaMissense is another Google DeepMind AI model that predicts the parts of the protein’s sequence where mutations (sites at which amino acids are incorrectly placed in the chain) could be dangerous or cancer-causing, and if the latter, the probability the cancer would be malignant rather than benign. These AlphaMissense predictions help the fight against cancer: if scientists learn which mutated proteins are cancer-causing, and what their unique structures are, [drugs could be better tailored to target those proteins while remaining unobtrusive to patients](https://www.bcrf.org/blog/ai-breast-cancer-drug-development).

## Looking forward
Google DeepMind has a reputation for its continued iteration and advancement of its innovative models. AlphaFold, on its third iteration now, still has room to grow. It is part of the burgeoning AI revolution in healthcare and biology \- a revolution that has the potential to completely change lives. 

However, this revolution did not start with AI, nor with the invention of mobile phones, or even the internet. In the field of molecular biology, the revolution began more than 50 years ago with the creation of the PDB, the open data infrastructure that facilitated all bioinformatics advances that followed.